### PR TITLE
New ham gg

### PR DESCRIPTION
**Pull request by PietroLiuzzo**
please note that this PR includes a split file, which is still experimental. I will use it to test functionality to support this.

### CODE REVIEW

File: Ham/DabraLibanosHamGG1/CCRsourceDesc.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <repository ref="INS0346DL"/>
+            <idno>CCR</idno>
+        </msIdentifier>
+        <msPart xml:id="p1CCR">
+            <!-- parte piccola, rilegata in un unico fascicolo-->
+            
+            <!--       perhaps   from here comes the error in the 
+              total added leaves by CCR, this is no quinion, 
+              is a senion and counts to 12 leaves.
+            -->
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/CCRsourceDesc.xml:14]: yes

File: Ham/DabraLibanosHamGG1/CCRsourceDesc.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            ff.1-2, 11-12 : Quiros + preghiere
+            <!--            foto bausi 1994-->
+            <msPart xml:id="p2CCR">
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/CCRsourceDesc.xml:17]: Qirqos (!)

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 3]
```diff
@@ -0,0 +1,1782 @@
...
+              1-1A-1C1C/0/1C-1C/C</ab>
+          </layout>
+        </layoutDesc>
+      </objectDesc>
+      <handDesc>
+        <handNote script="Ethiopic" xml:id="h8" corresp="#p1_i1">
+          <locus from="1" to="2"/>
+          <seg>The hand of the text on the verso of <locus target="2v"/>. No images are available of the previous folio or the recto of this.
+          </seg>
+          <seg type="rubrication">Some words rubricated.</seg>
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:99]: Personal names, numerals' stokes and punctuation marks

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 4]
```diff
@@ -0,0 +1,1782 @@
...
+        </handNote>
+        <handNote script="Ethiopic" xml:id="h13" corresp="#a25">
+          <locus from="12va" to="12vb"/>
+        </handNote>
+      </handDesc>
+    <additions>
+      <list>
+        <item xml:id="a26">
+          <locus from="11r" to="11r"/>
+          <!--          please Nafisa could you tell me what these are and if consistent with the rest of the text on this page?-->
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:116]: It is a separate note by a certain Gabra Hǝywat, it has nothing to do with what follows

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 5]
```diff
@@ -0,0 +1,1782 @@
...
+          <title ref="LIT6329CCR22"/>
+          <incipit><subst>
+            <del rend="erasure"/>
+            <add place="overstrike">አጽሐ</add>
+          </subst> ፍኩ፡ አነ፡ <persName ref="PRS1854Amdase">ዐምደ፡ ጽዮን፡ ንጉሥ። </persName>
+            በ፭፻፲ወ፪<sic>ዐመት፡ </sic>
+          </incipit>
+          <explicit>ውጉዘ፡ ለይኩን፡ በዝዓለም፡ ወበዘይመጽእ፡ ዓለም፡ ዘኢይትፈታሕ፡ በጾም፡ ወበ<del rend="erasure"/>ጸ<del
+            rend="erasure"/>
+            <pb/><cb/>ሎት፡ በስኢል፡ ወበስጊድ፡ ለዓለመ፡ ዓለም። <lb/><space/>
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:175]: I thik CCR 22 can be treated as one document ending up with a prayer, which is its part

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 6]
```diff
@@ -0,0 +1,1782 @@
...
+          <note>[4.22-26] = CCR 20: (I ff. 4vb-6vb), according to AB. </note>
+        </msItem>
+        <msItem xml:id="p2_i3">
+          <locus from="6vb16" to="7vb3"/>
+          <title ref="LIT6326CCR19"/>
+          <incipit>በ፭፻ወ፭፡ ዓመተ፡ ምሕረት፡ <pb/>ወሀብኩ፡ አነ፡ ዐምደ፡ ጽዮን፡ ንጉሥ፡ ለአባ፡ መጣዕ፡ ዘአሀም፡ </incipit>
+          <explicit>ከመ፡ ይትወሐስኒ፡ ፍቁርየ፡ መጣዕ፡ በመንግሥተ፡ ሰማያት፡ ለዓለመ፡ ዓለም፡ አሜን። </explicit>
+          <note>[4.26-28] = CCR 19: (I ff. 6vb-7vb), according to AB, thus the same comments are
+            applicabale to this part. I, however, decided to split what was considered one
+            document.</note>
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:208]: CCR 19 does not need to be splitted, I was too passionate

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 7]
```diff
@@ -0,0 +1,1782 @@
...
+          <locus from="7vb4" to="9ra3"/>
+          <title ref="LIT6305CCR7"/>
+          <incipit><sic>አጐልትኩ፡ </sic>አክሰምኩ፡ <sic>ወአመነይኩ፡ </sic>አነ፡ ሐፃኒ፡ <persName
+            ref="PRS6152Lalibala">ላሊበላ፡ </persName>ወስመ፡ መንግሥትየ፡ <persName
+              ref="PRS6152Lalibala">ገብረ፡ መስቀ ል፡ </persName>በ፬፻ወ፱። ዓመተ፡ ምሕረት፡ </incipit>
+          <explicit>ውጉዘ፡ ለይኩን፡ በቅድመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ በአፉሁ፡ ለአቡነ፡ መጣዕ፡ በአፈ፡ ጳጳሳት፡ በኣፈ፡ ፫፻፲፡
+            ወ፰፡ ርቱዐነ፡ ሃይማኖት፡ በአፈ፡ ፲ወ፭ነቢያት፡ በአፈ፡ ፲ወ፪ሐዋርያት፡ አሜን፡ ወአሜን። <lb/><space/></explicit>
+          <note>[4.28-30] = CCR 7: (I ff. 7vb-9ra), according to AB.</note>
+         
+<!--         this was listed twice, incipit is different, explicit also, same alignment to Bausi's list...-->
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:240]: Yes, because Bausi does treat them as one

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 8]
```diff
@@ -0,0 +1,1782 @@
...
+          <incipit>አግባእኩ፡ አነ፡ <persName ref="PRS6073Krestos">ክርስቶስ፡ አቡሁ፡ </persName>በአፈ፡
+            <persName ref="PRS1854Amdase">ዐምደ፡ ጽዮን፡ <title>ንጉሥ፡ </title></persName>ተርካ፡ ዘቀዳሚ፡
+            ምድሮሙ፡ ለቅዱሳን፡ </incipit>
+          <explicit>ወእመቦ፡ ዘደምሰሳ፡ ወዘቀደዳ፡ በጽልሑት፡ ከማሁ፡ ለይኩን፡ ውጉዘ፡ <sic>ለይኩንመ፡ ዓለመ፡ </sic>ዓለም፡ አሜን። </explicit>
+          <note>[4.30] = CCR 18: (I f. 9ra-b), according to AB.</note>
+        </msItem>
+        <msItem xml:id="p2_i6">
+          <locus from="9rb14" to="10rb18"/>
+          <title ref="LIT6340CCR33"/>
+          <incipit>አውሎግሶን፡ ጽደዙ፡ ዘአዘዙ፡ ንጉሥነ፡ ርቱዐ፡ ሃይማኖት፡ ሰይፈ፡ አርዓድ፡ ወስመ፡ መንግሥቱ፡ ቈስጠንጢኖስ፡ </incipit>
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:262]: ጽደቁ፡ is the second Ethiopic word

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 9]
```diff
@@ -0,0 +1,1782 @@
...
+        </physDesc>
+        <!--   f.20 to f.23     -->
+        <msPart xml:id="p7">
+          <msIdentifier>
+            <repository ref="INS0346DL"/>
+          </msIdentifier>
+          <msContents>
+            <msItem xml:id="p7_i3">
+              <locus from="20r2" to="20v12"/>
+<!--              what is on line 1?-->
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:681]: locus 20r1-20v12,
the incipit does not include the invocation as typical, but may be this needs to be changed

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 10]
```diff
@@ -0,0 +1,1782 @@
...
+                piange la morte, chiamandolo "colonna della fede". Nella citata storia delle sue
+                guerre, ed. Perruchon Ṣyon alle pagg. 10, 11, 16, 17, 30-31 sta cetramente a
+                designare Aksum, che ancor oggi è comunemente così chiamata. Degna di osservazione è
+                l'allusione alla poligamia del re, il che concorda pienamente con i dati della
+                storia in quistione. Non potrebbe poi passarsi sotto silenzio la diversità di
+                redazione fra gli atti di questo re e quelli di <persName ref="PRS6152Lalibala">re
+                  Lālibālā</persName></quote><bibl><ptr target="bm:ContiRossini1901Evangelodoro"/><citedRange unit="page">202–203</citedRange></bibl></cit>.</note>
+            </msItem>
+            <msItem xml:id="p7_i4">
+<!--              better locus needed, copied from previous msItem-->
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:702]: no need for splitting

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 11]
```diff
@@ -0,0 +1,1782 @@
...
+              <locus from="20r2" to="20v12"/>
+              <title>a request for a prayer about the king</title>
+              <incipit xml:lang="gez">ወአንትሙሂ፡ ቅዱሳን፡ እለ፡ ሀለውክሙ፡ ውስተ፡ መካነ፡ መጣዕ፡ ጸልዩ፡ ለንጉሥ፡ ዐምደ፡ ጽዮን፡ </incipit>
+              <explicit xml:lang="gez">ወበውስተ፡ ማኅበርሂ፡ ወኀበ፡ ተጋባእክሙሂ፡ ለዓለመ፡ ዓለም፡ አሜን። ። </explicit>
+              <note>[5.0] = CCR 19: (II f. 8r-v), according to AB, thus the same comments are
+                applicabale to this part. I, however, decided to split what was considered one
+                document.</note>
+            </msItem>
+            <msItem xml:id="p7_i6">
+              <locus from="20v13" to="22r8"/>
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:722]: 22v8

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 12]
```diff
@@ -0,0 +1,1782 @@
...
+              <locus from="22r9" to="22v4"/>
+              <title ref="LIT6327CCR20"/>
+              <incipit>ወለይእቲኒ፡ <persName ref="PRS2633BelenSa">ብሌን፡ ሰባ፡ <roleName type="title">በዐልተ፡
+                    ቢሐት፡ </roleName></persName>የሀባ፡ እግዚኣብሔር፡ ፍቅረ፡ ምስለ፡ <persName ref="PRS1854Amdase"
+                    ><roleName type="function">ምታ፡ </roleName>ዐምደ፡ ጽዮን፡ </persName></incipit>
+              <explicit>ወከመ፡ ኆጻ፡ ባሕር፡ አሜን፡ ወአሜን። ወይበል፡ ኵሉ፡ ሕዝብ፡ አሜን፡ ወአሜን። </explicit>
+              <note> It is still [5.2-6] = CCR 20: (II ff. 8v-10v), according to AB.</note>
+            </msItem>
+            <msItem xml:id="p7_i8">
+              <locus from="22v5" to="22v9"/>
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:763]: no need for splitting

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 13]
```diff
@@ -0,0 +1,1782 @@
...
+                      osservazione che per l'atto precedente: notisi come l'Agāmē qui apparisca
+                      distretto di poca importanza.</quote>
+                        <bibl><ptr target="bm:ContiRossini1901Evangelodoro"/>
+                          <citedRange unit="page">211–212</citedRange></bibl>
+                      
+                      </cit> </note>
+                  </q>
+                </item>
+                <item xml:id="a7">
+                  <!--                  what about ወአሜን at the end of line 18? it is not included in the explicit of p7_i11=LIT6325CCR18=CCR18-->
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:913]: I do not understand

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 14]
```diff
@@ -0,0 +1,1782 @@
...
+                writing is similar to that of n 20, but the hand is different, a little less trained
+                and slightly smaller. The last line is on page 15</quote></cit>.</note>
+            </msItem>
+            <msItem xml:id="p7_i11">
+              <locus from="23v2" to="23v18"/>
+              <title ref="LIT6325CCR18"/>
+              <incipit>አግባእኩ፡ አነ፡ <persName ref="PRS6073Krestos">ክርስቶስ፡ አቡሁ፡ </persName>በአፈ፡
+                  <persName ref="PRS1854Amdase">ዐምደ፡ ጽዮን፡ <title>ንጉሥ፡ </title></persName>ተርካ፡ ዘቀዳሚ፡
+                ምድሮሙ፡ ለቅዱሳን፡ </incipit>
+              <explicit>ወእመቦ፡ ዘደምሰሳ፡ ወዘቀደዳ፡ በጽልሑት፡ ከማሁ፡ ለይኩን፡ ውጉዘ፡ ለዓለመ፡ ዓለም፡ አሜን። </explicit>
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:797]: + ወአሜን።

File: Ham/DabraLibanosHamGG1/msDesc.xml (hunk 13 above)
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:913]: Yes, right point

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 15]
```diff
@@ -0,0 +1,1782 @@
...
+        </msPart>
+<!--  f.26-->
+        <msPart xml:id="p10">
+          <msIdentifier>
+            <repository ref="INS0346DL"/>
+          </msIdentifier>
+          <msContents>
+            <msItem xml:id="p10_i1">
+              <locus from="26r3" to="26r29"/>
+              <!--              I think this should be LIT6305CCR7, and there should be no second work record
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:1117]: it is a bit question whether it is one text or two. But I am ready to accept them as one. eshy. you won :-)

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 16]
```diff
@@ -0,0 +1,1782 @@
...
+                sentimenti religiosi, fosser promosse da ragioni politiche e intese ad accaparrare
+                al governo del Lasta il favore del Tigrè settentrionale</quote>
+                  <bibl><ptr target="bm:ContiRossini1901Evangelodoro"/>
+                    <citedRange unit="page">192–193</citedRange></bibl>
+                </cit>. Roger Schneider noted:
+                '<cit><quote xml:lang="fr">Photo p.4. Au-dessus tresse formée de deux fils en roulis</quote></cit>'.</note>
+            </msItem>
+            <msItem xml:id="p10_i3">
+              <locus from="26v16"/>
+<!--              not sure about this placement... the transcription is all messed up, the image hard to read.
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:1176]: yes from line 16 to the end of the page, I count to 24, but not sure either.

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 17]
```diff
@@ -0,0 +1,1782 @@
...
+                </subst> ንጉሥ። በ፭፻፲ወ፪። ዓመተ፡ ምሕረት፡ </incipit>
+              <explicit>የሀቦ፡ እግዚአብሔር፡ ጥበበ፡ ከመ፡ ሰሎሞን፡ አሜን። የሀቦ፡ እግዚኣብ </explicit>
+              <note>[5.16-20] = CCR 22: (II ff. 16ra-17vb), according to AB.</note>
+           
+<!--           newly identified by Nafisa? not bausi list, part of the previous (nested into it)-->
+            <msItem xml:id="p12_i1.1">
+              <locus from="29va17" to="29vb24"/>
+              <title type="incomplete">a prayer about the king</title>
+              <incipit>ወለንጉሥነ፡ <subst>
+                  <del rend="erasure"/>
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:1330]: Nafisa's bad idea

File: Ham/DabraLibanosHamGG1/msDesc.xml [hunk 18]
```diff
@@ -0,0 +1,1782 @@
...
+              <handNote xml:id="h35" script="Ethiopic" corresp="#a13">
+                <locus from="30v24"  to="30v28"/>
+                <seg>CCR dated this hand to the XVI century.</seg>
+              </handNote>
+              
+            </handDesc>
+            <additions>
+              <list>
+                <item xml:id="a13">
+<!--                  not sure about the placement of this, expecially given the explicit of the previous item and the del markup which is mysteriuous-->
```
Comment (nafisa-valieva) [inline Ham/DabraLibanosHamGG1/msDesc.xml:1422]: it starts on line 22 to 28

**Comment by nafisa-valieva** [comment]
Ciao Pietro,
Great work! I have made some comments while reading the text, I hope it is fine. I see some mistakes in my own transcription, but I think I prefer to check everything at the very end of our work, may be seeing it online and having new layout and new impressions in general. I have not checked the text transcribed with Transcribus, do you want me to do it?

Yours,
Nafisa

**Review by nafisa-valieva — CHANGES_REQUESTED** [review]
Dear Pietro,
great work! here my comments on our file (I went through others as well, but I understand very little)
In general I see some errors in my transcription, I would like to revise them when we are close to the end. I also want to check hands identifications once again.
Quiros is Qirqos,
CCR 19 and CCR 20, CCR 22 in both sets should be reunited, it was not a good idea, and I also give up upon my ideas on CCR 6 a and b and 7a and b :-) 
concerning locus and incipits: as a general practice I have skipped the invocation formula in the incipits, if you think it is confusing, I can add it, it is easy. But as a result, it affected your understanding of locus sometimes, so I noted it.
Yours, Nafisa

**Review by PietroLiuzzo — COMMENTED** [review]
Dear Nafisa, for the transcription checking, let us see if it is worth, for now I would think it is not a priority.

**Comment by PietroLiuzzo** [comment]
ok, I will make the necessary changes. thanks!

**Comment by PietroLiuzzo** [comment]
@nafisa-valieva I think I have done what needed doing for CCR6, 7, 19, 20 and 22. I am not sure about the loci. I think, technically speaking we need to be inclusive with those, they need to say where a text starts, inclusive of any formulaic element. it would be very interesting to mark in the transcription or in the work record, using the EMA methodology, the various parts of the documents.

**Comment by PietroLiuzzo** [comment]
@nafisa-valieva , I think this is ready to go. if you are ok with it I will proceed to merge https://github.com/BetaMasaheft/Works/pull/460 and then this one, I need to do this myself because I updated the syncing script and i need to observe and test using this PR.

**Comment by PietroLiuzzo** [comment]
Dear @nafisa-valieva the test has gone well for the step being tested, but please, do not worry too much about the current, broken, view of the manuscript online. because it has new encoding, it has been submitted in a new way and as part of a major rearchitecturing of the backend, it is behaving better than expected. with the new release this will go fine, for the time being I will simply do a work around.

**Comment by nafisa-valieva** [comment]
Ok! Thank you for this word of warning :-) For now I cannot see anything online, it gives only error.

**Comment by PietroLiuzzo** [comment]
well, there are a couple of errors which I can also see but "only error" is perhaps a bit too much? what are you experiencing?

**Comment by nafisa-valieva** [comment]
iiii! now I see it too!

**Pull request merged**